static/scss/videoDetail.scss

```diff
@@ -24,3 +24,73 @@
     }
   }
 }
+
+.videoSubtitles {
```

Comment:
Let's change `videoSubtitles*` to `video-subtitles*`. We don't do this mixed camel case + dashes naming anywhere else in the css

Reply:
OK